### PR TITLE
improve: reduce errors in relayer tests

### PR DESCRIPTION
**Pull request by james-a-morris**

### CODE REVIEW

File: test/Dataworker.loadData.ts [hunk 1]
```diff
@@ -729,7 +728,10 @@ describe("Dataworker: Load data used in all functions", async function () {
     );
     const blockNumber = await spokePool_2.provider.getBlockNumber();
     const blockTimestamp = (await spokePool_2.provider.getBlock(blockNumber)).timestamp;
-    const realizedLpFeePctData = await hubPoolClient.computeRealizedLpFeePct(deposit1, l1Token_1.address);
+    const realizedLpFeePctData = await hubPoolClient.computeRealizedLpFeePct(
+      { ...deposit1, blockNumber },
```
Comment (mrice32) [inline test/Dataworker.loadData.ts:732]: Is this a functional change?

File: test/utils/utils.ts [hunk 2]
```diff
@@ -272,138 +286,43 @@ export async function simpleDeposit(
   };
 }
 
-export async function deploySpokePoolForIterativeTest(
```
Comment (mrice32) [inline test/utils/utils.ts:275]: Were all of these unused?

**Review by mrice32 — APPROVED** [review]
Looks good! Just want to double check that there are no functional changes here, and this is only a refactor. Is that the intention?

File: test/utils/utils.ts (hunk 2 above)
Comment (james-a-morris) [reply test/utils/utils.ts:275]: Yep!

File: test/Dataworker.loadData.ts (hunk 1 above)
Comment (james-a-morris) [reply test/Dataworker.loadData.ts:732]: This PR is related to refactorings & slight rewrites to get the software to be able to compile with tsc. 

Prior to this, we were having actual compiler errors due to signature incongruence on function/constructor calls.

**Comment by james-a-morris** [comment]
@mrice32 resolving as we discussed offline.

**Pull request merged**